packages/codegen/src/generateEntityCodegenFile.ts

```diff
@@ -205,7 +211,7 @@ export function generateEntityCodegenFile(config: Config, meta: EntityDbMetadata
       ${[o2m, m2o, m2m]}
 
       constructor(em: ${EntityManager}, opts: ${entityName}Opts) {
-        super(em, ${metadata});
+        ${hasDefaultValues ? code`super(em, ${metadata}, {...${defaultValuesName}})` : code`super(em, ${metadata})`};
```

Comment:
Cool approach with the `...`

Reply:
I was initially just passing the DefaultValues object directly, which broke everything because every entity of a class ended up having the same object for `__orm.data` 😂